### PR TITLE
Don't calculate coverage on the spec directory

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -27,6 +27,7 @@
   add_filter 'lib/traject/tei_config.rb'
```
Comment (cbeer) [inline spec/spec_helper.rb]: I think if we re-add `SimpleCov.start('rails') do` above, we'll get these normal filters for free, no?

Comment (jcoyne) [inline spec/spec_helper.rb]: @cbeer I tried that. When doing that, it disregards all the filters in the block.

Comment (jcoyne) [inline spec/spec_helper.rb]: See https://coveralls.io/builds/12862044

**Comment by cbeer** [comment]
I think #87 is probably a better approach, and I'll continue to follow-up with the simplecov maintainers to see if there's something better we can do.

**Pull request closed without merging**